src/main/java/com/conveyal/r5/profile/ProfileRequest.java

```diff
@@ -63,7 +63,7 @@
     public int bikeTrafficStress = 4;
     
     /** The speed of driving, in meters per second. Roads from OSM use the speed limit, this is the speed used when propagating from the street network to a pointset. */
-    public float carSpeed = 11; // ~40 km/h
+    public float carSpeed = 2.22f; // ~8 km/h
```

Comment:
I believe this is the off-road driving speed, which is usually a access road / parking lot speed, so 8km/h makes sense. But is this change intentionally bundled with this batch of linking fixes?